django_cloud_deploy/skeleton/source_generator.py

```diff
@@ -729,6 +837,7 @@ def generate_from_existing(self,
             image_tag, cloudsql_secrets, django_secrets)
         self.app_engine_file_generator.generate_from_existing(
             project_name, project_dir, service_name)
+        self.install_requirements(project_dir)
```

Comment:
Why do we need to do this?

Reply:
This is because our tool runs code of the existing Django project. For example, when migrating database and loading settings files. This may need those dependencies.